invenio_rdm_records/records/systemfields/access/field.py

```diff
@@ -86,6 +95,11 @@ def grants(self):
         """The set of permission grants for the record."""
         return self._grants
 
+    @property
+    def links(self):
+        """The set of secret links for the record."""
+        return self._links
```

Comment:
question: could we just save `self.links` (not `self._links`) directly and save us this?

Reply:
hmm... i'm not 100% sure about my own reasoning here from back then anymore :thinking: 
the only upside of this `self._x` pattern here that i can think of right now is that it safeguards against unintended direct assignments (e.g. `record.access.links = 5`), which could break things
if that's not an issue though, the code would get much more concise if we did away with that.
your call :smiley_cat:

Reply:
This `e.g. record.access.links = 5` was also the only case I saw but wouldn't really happen (as in: code will break pretty quickly after this anyway and we would never parry all cases due to nature of the language). Was just wondering about it - i.e. if I don't *need* `property` I don't go for it so I was curious.